meetings/2022-04-28.md

```diff
@@ -53,7 +56,11 @@ URL: https://github.com/solid/notifications/pull/59
 
 * SC: Minor follow-up. Not urgent. Time to integrate this in while reviews from the Editors team is taking place. Can we wrap this up this week so that we can include it part of Notifications Protocol that's currently under review ( https://github.com/solid/specification/pull/391 )?
 * SC: Made a brief comment to move this forward; otherwise we can merge the suggestions. The text between README and spec does not need to be different.
```

Comment:
```suggestion
* SC: Made a brief comment to move this forward; otherwise we can merge the suggestions. The text does not need to be different between README and spec.
```